src/types.rs

```diff
@@ -35,6 +35,8 @@ pub struct RustInfo {
     pub target_pointer_width: Option<String>,
     /// target vendor cfg value
     pub target_vendor: Option<String>,
+    /// target triple cfg value
```

Comment:
its not really a cfg value, its constructed. so maybe document it as that: 'target triple constructed from target arc, vendor, os and env'

Reply:
Done.